src/diff.sh

```diff
@@ -50,6 +50,11 @@ function diff_manager()
     return 0
   fi
 
+  if [[ -d "$target_1" && -d "$target_2" ]]; then
+    diff_folders "$target_1" "$target_2"
+    return 0
```

Comment:
`diff_folders` might return a failure status code, also return it instead of return 0; i.e., `return "$?"`.